pset-iterators.js

```diff
@@ -115,6 +136,21 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+
+const presidentsBetween1850and1900 = presidents.filter(function (objectPresident) {
+  const yearTookOffice = parseInt(objectPresident.took_office.substring(0,4));
+
+     if (yearTookOffice > 1850 && yearTookOffice < 1900) {
+        return true;
+     }
```

Comment:
Great use the .filter method here; keep in mind, that filter only returns elements what match the "true" condition; that is, if it is false it won't return it (so no need to specify that in the else block). All you need is what will satisfy the true condition on this method.